### PR TITLE
Druid tests

### PR DESCRIPTION
**Pull request by fhennig**
## Review Checklist
- [x] Code contains useful comments
- [x] Changelog updated (or not applicable)

### CODE REVIEW

**Comment by fhennig** [comment]
- All the checks passed
- I added comments
- There doesn't seem to be a changelog, so I considered that as 'not applicable'

I tested with the dev-cluster.yml kind config that is checked into the druid-operator repository.
A running zookeeper cluster with the name 'simple' is required for this test.

File: druid-operator/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/target
+Cargo.lock
```
Comment (razvan) [inline druid-operator/.gitignore:2]: Cargo.lock is added to .gitignore but also already added to git.

File: druid-operator/tests/common/checks.rs [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (razvan) [inline druid-operator/tests/common/checks.rs:1]: Is this file needed?

File: druid-operator/tests/common/mod.rs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+pub mod checks;
```
Comment (razvan) [inline druid-operator/tests/common/mod.rs:1]: This module is empty. I would remove it.

**Review by razvan — COMMENTED** [review]
I would add a README.md with the requirements (zk, kind, etc.)

File: druid-operator/.gitignore (hunk 1 above)
Comment (fhennig) [reply druid-operator/.gitignore:2]: I don't see it added, are you sure? I don't see it in the branch on github and not on my machine if I pull the latest version

**Review by razvan — APPROVED** [review]
LGTM
